### PR TITLE
feat: response log detail

### PR DESCRIPTION
**Pull request by sebastianmraz**

### CODE REVIEW

**Comment by matusklasovity** [comment]
@andrej-jasso @sebastianmraz tento a [tento](https://github.com/GoodRequest/AppDebugMode-iOS/pull/27) PR su si akosi velmi podobne 😄

File: Sources/AppDebugMode/Screens/ConsoleLogsVIew/ConsoleLogsView.swift [hunk 1]
```diff
@@ -36,33 +37,40 @@ struct ConsoleLogsView: View {
...
+
+                if isLoading {
+                        ProgressView()
+                            .progressViewStyle(.circular)
+                            .progressViewStyle(CircularProgressViewStyle(tint: AppDebugColors.primary))
+                            .frame(maxWidth: 20, maxHeight: 20, alignment: .center)
+
+                        Color.gray.opacity(0.5)
+                            .frame(maxWidth: .infinity, maxHeight: .infinity)
+                            .ignoresSafeArea()
```
Comment (matusklasovity) [inline Sources/AppDebugMode/Screens/ConsoleLogsVIew/ConsoleLogsView.swift:69]:
```suggestion
                    Color.black.opacity(0.9)
                        .frame(maxWidth: .infinity, maxHeight: .infinity)
                        .ignoresSafeArea()

                    ProgressView()
                        .progressViewStyle(CircularProgressViewStyle(tint: AppDebugColors.primary))
                        .frame(maxWidth: 20, maxHeight: 20, alignment: .center)
```

**Review by matusklasovity — CHANGES_REQUESTED** [review]

**Comment by matusklasovity** [comment]
Pokial kliknem na fetch large tak sa spinner donekonecna toci
Snímka obrazovky 2024-03-05 o 8 16 24

**Comment by matusklasovity** [comment]
Ked uz si sa hral s navigaciou mohol by si sa pohrat aj s navigation barom. Nechsu tie buttony troska viac citatelne ako teraz.
Snímka obrazovky 2024-03-05 o 8 17 20

**Comment by matusklasovity** [comment]
Snímka obrazovky 2024-03-05 o 8 18 15

**Review by matusklasovity — APPROVED** [review]

**Review by andrej-jasso — APPROVED** [review]

**Pull request merged**